app/win/record.js

```diff
@@ -0,0 +1,37 @@
+// Application state fallback upon unexpected quit
+const {app} = require('electron');
+const Config = require('electron-config');
+const wins = require('./windows');
+
+// local storage
+const rec = new Config();
+const recordInterval = 2000;
```

Comment:
You can't lower this cause the process it take to save when there is multiple windows open with multiple panes